code/style.css

```diff
@@ -2,26 +2,32 @@
   box-sizing: border-box;
 }
 
-body {
+.body {
   margin: 0;
   padding: 0;
-  font-family: 'Montserrat', sans-serif;
-  background: #0026ff;
+  background: #f3cbe7;
 }
 
-h1 {
+#name-input {
+  background: #fcedf7;
+}
```

Comment:
You should avoid to styling your code with id selector. This can provide a lot of complications in bigger apps. You can always create a class with the same name as the id 😸